spring-boot/src/main/java/org/springframework/boot/SpringApplication.java

```diff
@@ -353,6 +361,10 @@ private void prepareContext(ConfigurableApplicationContext context,
 		// Add boot specific singleton beans
 		context.getBeanFactory().registerSingleton("springApplicationArguments",
 				applicationArguments);
+		String bannerBeanName = environment.getProperty(BANNER_BEAN_NAME_PROPERTY, String.class, "springBanner").trim();
```

Comment:
We should just do it by default, something like `springBootBanner` (see what was done in `LoggingApplicationListener` for the `LoggingSystem`).